records/board/mimxrt1170_evk_qspi.yaml

```diff
@@ -0,0 +1,13 @@
+common:
+    macros:
+        - SOFT_RESET=VECTRESET
+        - SWO_UART=1
+        - SWO_USART_PORT=1
```

Comment:
I don't know that it matters, but I have been putting the `SWO_UART_*` macro definitions in `DAP_config.h`.